po/zh_CN.po

```diff
@@ -1706,7 +1704,7 @@ msgstr "欢迎来到 GnuCash.org"
 #: index.phtml:15
 #, php-format
 msgid "Download GnuCash %s"
-msgstr "下载 GnuCash %s"
+msgstr "下载 GnuCash"
```

Comment:
@duguqiubailee  %s is a string parameter, which later gets substituted by some real content. Please see https://wiki.gnucash.org/wiki/Translation#Special_characters_and_other_tips